fixtures/cameo/zenit-z120.json

```diff
@@ -0,0 +1,270 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Zenit Z120",
+  "shortName": "Z120",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["EP"],
+    "createDate": "2020-06-11",
+    "lastModifyDate": "2020-06-11"
+  },
+  "links": {
+    "productPage": [
+      "https://www.cameolight.com/fr/solutions/integrateurs/eclairage-statique/projecteurs-a-par/2416/zenit-z120?c=2038#detail-downloads",
+      "https://www.cameolight.com/fr/solutions/integrateurs/eclairage-statique/projecteurs-a-par/2416/zenit-z120?c=2038#detail-downloads",
+      "https://www.cameolight.com/fr/solutions/integrateurs/eclairage-statique/projecteurs-a-par/2416/zenit-z120?c=2038#detail-downloads"
```

Comment:
This URL is dead.  I think the "Z120 G2" superseded the "Z120", and Cameo wiped all evidence of the "Z120" from their website.

I found a copy of the "Z120" manual at: https://www.adamhall.com/shop/gr-en/downloads/file/id/864775656